mod/activity/elgg-plugin.php

```diff
@@ -19,4 +19,13 @@
 			'context' => ['profile', 'dashboard'],
 		],
 	],
+	'routes' => [
+		'activity' => [
+			'path' => '/activity/{page_type}/{username?}',
```

Comment:
I think better to have multiple routes. Even though same view can handle them, different logical resources.

Reply:
Sure